deploy/examples/README.md

```diff
@@ -0,0 +1 @@
+# Examples has moved under [/examples](../../examples)
```

Comment:
Could you make this normal text, rather than a title, and change to
"NOTE: Please note that most of the examples have now been relocated to [/examples](../../examples)."